lib/std/bounded_array.zig

```diff
@@ -401,6 +396,14 @@ test "BoundedArray sizeOf" {
     try testing.expectEqual(@TypeOf(@as(BoundedArray(u8, 16), undefined).len), u5);
 }
 
+test BoundedArrayAligned {
+    var a = try BoundedArrayAligned(u8, 16, 2).init(0);
```

Comment:
```suggestion
/// var a = BoundedArrayAligned(u8, 16, 2){};
```
https://github.com/ziglang/zig/pull/16993#discussion_r1306728192